packages/cli/create-strapi-starter/package.json

```diff
@@ -45,6 +53,10 @@
     "inquirer": "8.2.5",
     "ora": "5.4.1"
   },
+  "devDependencies": {
+    "eslint-config-custom": "*",
+    "tsconfig": "*"
```

Comment:
Is there a reason for not fixing these at at least a major version?

Reply:
Yes those are internal dependencies of the monorepo so it will just link them locally there is no need for a version (packages being private here they never get published)